src/main/java/tr/com/srdc/cda2fhir/transform/section/impl/SectionResultSingular.java

```diff
@@ -0,0 +1,27 @@
+package tr.com.srdc.cda2fhir.transform.section.impl;
+
+import java.util.List;
+
+import org.hl7.fhir.dstu3.model.Bundle;
+import org.hl7.fhir.dstu3.model.Resource;
+
+import tr.com.srdc.cda2fhir.util.FHIRUtil;
+
+public class SectionResultSingular<T extends Resource> extends SectionResult {
+	private Class<T> clazz;
+	
+	public SectionResultSingular(Bundle bundle, Class<T> clazz) {
+		super(bundle);
+		this.clazz = clazz;
+	}
+
+	public static <U extends Resource> SectionResultSingular<U> getInstance(Bundle bundle, Class<U> clazz) {
```

Comment:
Just curious: Why the use of a static function instead of just the constructor?

Reply:
It looked easier to call with generic template variables.